MetaMask/metamask-mobile
# fix: Update Basic Functionality settings description.json

## **Description**
Adds copy to explain:
- What happens when toggle is enabled/disabled
- Basics of internet services and exposing IP addresses (we’re not doing anything out of the ordinary)
- IP address use by us is temporary and limited
- We never sell user data

## **Related issues**
Fixes: https://github.com/MetaMask/MetaMask-planning/issues/2560

## **Manual testing steps**
Onboarding
1. Go to Onboarding > Default Settings
2. Check copy under Basic Functionality
3. Check link to Privacy Policy

Settings
1. Go to Setttings > Privacy and Security
2. Check copy under Basic Functionality
3. Check link to Privacy Policy

## **Screenshots/Recordings**


#table markdown template
| Before       | After        |
|--------------|--------------|
| <img width="350" alt="Screenshot 2024-04-18 at 3 56 43 PM" src="https://github.com/MetaMask/metamask-mobile/assets/165834542/07e7e4fe-a97d-4244-b789-cf235171857d"> |<img width="350" alt="Screenshot 2024-04-18 at 3 56 43 PM" src="https://github.com/MetaMask/metamask-mobile/assets/165834542/23c9aaf2-16c2-4ba2-b1ee-8eb9016b2dbe"> |

### **Before**



### **After**



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
